horizons/constants.py

```diff
@@ -635,7 +635,7 @@ class NETWORK:
 	# change port to 2022 for development server updated after UH commits
 	SERVER_PORT = 2002
 	CLIENT_ADDRESS = None # type: Optional[str]
-	UPDATE_FILE_URL = "http://updates.unknown-horizons.org/current_version.php"
+	UPDATE_FILE_URL = "http://unknown-horizons.org/current-version.json"
```

Comment:
It doesn't. The certificate for that domain is for `*.github.com`. According to [this](https://github.com/blog/2186-https-for-github-pages) GitHub Pages supports HTTP, but only for user pages. If I try and access `https://unknown-horizons.github.io/current-version.json` I get redirected to `http://unknown-horizons.org/current-version.json`.